src/Controller/MemberController.php

```diff
@@ -40,6 +42,17 @@ public function showOneEventById(Event $event): Response
     }
 
     /**
+     * @param UserRepository $userRepository
+     * @Route("/page/{id}", name="member_page")
+     * @return Response
+     */
+    public function index(UserRepository $userRepository, User $user) : Response
```

Comment:
c'est une page accessible à tout le monde ? un profil "public" ?
car si non, il ne faut pas faire comme ça. Dans tous les cas, le user est récup via le paramconverter donc pas besoin du UserRepository, ni du findOneBy en dessous, tu as déjà ton objet user
si c'est pas "public" le profil, un $this->getUser() est suffisant

Reply:
Je viens de corriger, du coup pour sécuriser les comptes pour qu'un utilisateur ne puisse pas avoir acces au compte d'un autre, cela se fera directement via le secure des routes, vu avec Bertrand ! :)